tensorbay/opendataset/BDD100K/loader.py

```diff
@@ -279,6 +361,73 @@ def _merge_label(source_label_contents: List[List[Dict[str, Any]]]) -> Dict[str,
     return label_contents
 
 
+def _get_instance_mask(filename: str, original_mask_path: str, mask_directory: str) -> InstanceMask:
+    mask_path = os.path.join(mask_directory, f"{filename}.png")
```

Comment:
can we use `mask_filename` rather than `mask_path`?

Reply:
this is actually a path, use `mask_filename` may be a little weird.